### PR TITLE
SEQNG-1045 Fixed bug that prevented to load non calibration sequences.

### PR DESCRIPTION
**Pull request by jluhrs**

### CODE REVIEW

File: modules/seqexec/server/src/main/scala/seqexec/server/SeqTranslate.scala [hunk 1]
```diff
@@ -353,17 +353,11 @@ class SeqTranslate(site: Site, systems: Systems[IO], settings: TranslateSettings
     sys: InstrumentSystem[IO]
   ): TrySeq[List[System[IO]]] = {
     stepType match {
-      case StepType.CelestialObject(inst) => for {
-        tcs  <- getTcs(inst.hasOI.fold(allButGaos, allButGaosNorOi), useGaos = false, sys,
-          TcsController.LightSource.Sky, config)
-        gcal <- Gcal.fromConfig(systems.gcal, site == Site.GS)(config)
-      } yield List(sys, tcs, gcal)
+      case StepType.CelestialObject(inst) => getTcs(inst.hasOI.fold(allButGaos, allButGaosNorOi), useGaos = false, sys,
```
Comment (cquiroz) [inline modules/seqexec/server/src/main/scala/seqexec/server/SeqTranslate.scala]: There are a bit hard to read, could we perhaps write the `.map` in a different line and the parametrs for `getTcs` each on its own line?

File: modules/seqexec/server/src/main/scala/seqexec/server/SeqTranslate.scala [hunk 2]
```diff
@@ -353,17 +353,11 @@ class SeqTranslate(site: Site, systems: Systems[IO], settings: TranslateSettings
...
+          TcsController.LightSource.Sky, config).map{ List(sys, _, Gcal.defaultGcal(systems.gcal)) }
 
-      case StepType.NodAndShuffle(inst)   => for {
-        tcs  <- getTcs(inst.hasOI.fold(allButGaos, allButGaosNorOi), useGaos = false, sys,
-          TcsController.LightSource.Sky, config)
-        gcal <- Gcal.fromConfig(systems.gcal, site == Site.GS)(config)
-      } yield List(sys, tcs, gcal)
+      case StepType.NodAndShuffle(inst)   => getTcs(inst.hasOI.fold(allButGaos, allButGaosNorOi), useGaos = false, sys,
+          TcsController.LightSource.Sky, config).map{ List(sys, _, Gcal.defaultGcal(systems.gcal)) }
```
Comment (cquiroz) [inline modules/seqexec/server/src/main/scala/seqexec/server/SeqTranslate.scala]: Why don't you need `site` anymore?

Comment (jluhrs) [reply modules/seqexec/server/src/main/scala/seqexec/server/SeqTranslate.scala]: There is a difference in when the IR lamp is turned on at both sites, but it does not matter when GCAL is not used.

File: modules/seqexec/server/src/main/scala/seqexec/server/SeqTranslate.scala (hunk 1 above)
Comment (jluhrs) [reply modules/seqexec/server/src/main/scala/seqexec/server/SeqTranslate.scala]: Done

**Pull request merged**